tests/Responses/Expected/product/create_with_associations_response.json

```diff
@@ -0,0 +1,167 @@
+{
+    "id": @integer@,
+    "name": "Theme Mug",
+    "code": "MUG_TH",
+    "available_on": "@string@.isDateTime()",
+    "attributes": [],
+    "variants": [],
+    "options": [],
+    "associations": [
+        {
+            "id": @integer@,
+            "type": {
+                "id": @integer@,
+                "code": "similar",
+                "created_at": "@string@.isDateTime()",
+                "updated_at": "@string@.isDateTime()",
+                "translations": [
+                    {
+                      "id": @integer@,
+                      "locale": "en"
+                    }
+                ],
+                "current_locale": "en",
+                "fallback_locale": "en_US"
+            },
+            "associated_products": [
+                {
+                    "id": @integer@,
+                    "name": "Star wars mug",
+                    "code": "MUG_SW",
+                    "available_on": "@string@.isDateTime()",
+                    "attributes": [],
+                    "variants": [],
+                    "options": [],
+                    "associations": [],
+                    "translations": {
+                        "en": {
```

Comment:
Ok, now I see that you use `en` and `en_US` as two different locales... It's correct, but maybe we can use something a little bit more various? `en_US` and `pl_PL`? It's just confusing if you see two translations with almost the same values 😕